src/pages/ReimbursementAccount/ContinueBankAccountSetup.tsx

```diff
@@ -12,88 +10,85 @@ import ScreenWrapper from '@components/ScreenWrapper';
 import ScrollView from '@components/ScrollView';
 import Section from '@components/Section';
 import Text from '@components/Text';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import WorkspaceResetBankAccountModal from '@pages/workspace/WorkspaceResetBankAccountModal';
 import * as BankAccounts from '@userActions/BankAccounts';
 import CONST from '@src/CONST';
-import * as ReimbursementAccountProps from './reimbursementAccountPropTypes';
+import type * as OnyxTypes from '@src/types/onyx';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
 
-const propTypes = {
+type ContinueBankAccountStepProps = {
     /** Bank account currently in setup */
-    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountPropTypes.isRequired,
+    reimbursementAccount: OnyxEntry<OnyxTypes.ReimbursementAccount>;
 
     /** Callback to continue to the next step of the setup */
-    continue: PropTypes.func.isRequired,
+    onContinue: () => void;
```

Comment:
perhaps `onContinuePress` would fit better here, since we also have `onBackButtonPress`